release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -29,6 +30,20 @@
 
 yaml_path = pathlib.Path("/tmp/ray_v1alpha1_rayservice.yaml")
 
+KILL_WORKER_NODE = "kill_worker_node"
+KILL_HEAD_NODE = "kill_head_node"
+
+
+def generate_cluster_variable():
```

Comment:
```suggestion
def generate_cluster_configs():
```

Reply:
I will keep it as variable, config is not quite accurate, config generate is under difference function